autoconf-tester/.coveragerc

```diff
@@ -0,0 +1,10 @@
+[report]
```

Comment:
@slowr could you comment here how we should exactly deal with this file? And what should be done with coverage conversion, since these are parallel tasks? So that we also deal with this with this PR once and for all. Thanks!

Reply:
they are parallel tasks but in seperate machines so we don't have an issue with file replaces or whatever